server/proxy/service.go

```diff
@@ -125,9 +127,6 @@ func (p *Proxy) CanDeploy(deployType apps.DeployType) (allowed, usable bool) {
 	return p.canDeploy(p.conf.Get(), deployType)
 }
 
-// CanDeploy returns the availability of deployType.  allowed indicates that the
-// type can be used in the current configuration. usable indicates that it is
-// configured and can be accessed, or deployed to.
```

Comment:
Why was this comment removed?

Reply:
It was redundant with `p.CanDeploy`, but I added it back.